homeassistant/components/venstar/sensor.py

```diff
@@ -160,15 +178,15 @@ def native_value(self) -> int:
     @property
     def native_unit_of_measurement(self) -> str | None:
         """Return unit of measurement the value is expressed in."""
-        return self.entity_description.uom_fn(self.coordinator)
+        return self.entity_description.uom_fn(self.coordinator, self.sensor_name)
```

Comment:
I think we can do this smarter, but let's await the other question first

Reply:
To describe my issue with this btw, let's add a way to do static unit of measurements without adding all kinds of untyped parameters to a function.

Something like the default implementation would work

```py
if hasattr(self.entity_description, "native_unit_of_measurement"):
    return self.entity_description.native_unit_of_measurement
```
If you add this as the first line of the function, you could have 2 entity descriptions for the new entities and just set `native_unit_of_measurement`.